modules/tfe-instance/aws.tf

```diff
@@ -307,6 +311,8 @@ EXTRA_NO_PROXY="${var.no_proxy}"
 }
 
 resource "aws_elb" "ptfe" {
+  name_prefix = "${var.elb_name_prefix}"
+
```

Comment:
It's nothing much.

But you could delete this empty line, just to follow the module standard.

Reply:
Done. FWIW the module standard seems to be quite inconsistent, there are newlines after the name argument [here](https://github.com/hashicorp/terraform-enterprise-modules/blob/master/modules/tfe-instance/iam.tf#L2) and [here](https://github.com/hashicorp/terraform-enterprise-modules/blob/master/modules/tfe-instance/aws.tf#L225) for example.